src/gui/properties/propertieswidget.cpp

```diff
@@ -173,6 +175,39 @@ PropertiesWidget::~PropertiesWidget()
     delete m_ui;
 }
 
+void PropertiesWidget::displayToggleColumnsMenu(const QPoint &)
```

Comment:
Columns of what?
I would just indicate which control the menu belongs to.
`displayFileListHeaderMenu()` is better, IMO.